src/rtcmix/Makefile

```diff
@@ -26,6 +26,8 @@ CONFDIR = ../conf
 CXXFLAGS += $(SOUND_DRIVER)
 INCLUDES += -I$(INCLUDEDIR) -I../include -I$(AUDIODIR) -I$(SNDLIBDIR) -I./heap -I../conf
 
+INCLUDES += -I../osc
+INCLUDES += ../osc/RTOSCListener.o
```

Comment:
This is not an INCLUDE - this is an addition to the set of objects being compiled and linked.
Note the list of XXX_OBJS at the top of the Makefile.  Use it instead of tINCLUDES here.

Reply:
This still needs to be addressed.